docs/TinyAra_Config_Variables.html

```diff
@@ -2498,8 +2498,6 @@ <h1>Table of Contents</h1>
 <li><a href="#CONFIG_SMARTFS_MULTI_ROOT_DIRS">1.10.9.3 <code>CONFIG_SMARTFS_MULTI_ROOT_DIRS</code>: Support multiple Root Directories / Mount Points</a></li>
 <li><a href="#CONFIG_SMARTFS_ALIGNED_ACCESS">1.10.9.4 <code>CONFIG_SMARTFS_ALIGNED_ACCESS</code>: Ensure 16 and 32 bit accesses are alined</a></li>
 <li><a href="#CONFIG_SMARTFS_DYNAMIC_HEADER">1.10.9.6 <code>CONFIG_SMARTFS_DYNAMIC_HEADER</code>: Dynamic Header</a></li>
-<li><a href="#CONFIG_SMARTFS_JOURNALING">1.10.9.7 <code>CONFIG_SMARTFS_JOURNALING</code>: Enable filesystem journaling for smartfs</a></li>
```

Comment:
we can keep CONFIG_SMARTFS_JOURNALING but not CONFIG_SMARTFS_JOURNAL_VERIFY

Reply:
Done